controllers/user.controller.js

```diff
@@ -76,13 +76,11 @@ class UserController {
           email: user.email,
         };
         const issueToken = helper.generateToken(user.dataValues);
-        res.status(201).send({
+        return res.status(201).send({
           status: 201, message: 'User added successfully', token: issueToken, user: userData
         });
       }
-      if (err) {
-        res.status(409).send({ status: 409, message: err.parent.detail });
-      }
+      return res.status(500).send({ status: 500, message: 'Internal Server Error' });
```

Comment:
What is the reason for return internal server error?

Reply:
I have validated all expected error from this endpoint, So else I'm expecting HTTP status 500 for bad scripting error ( internal server error )